index.html

```diff
@@ -21,6 +21,17 @@ <h2>
       </button>
     </form>
 
+    <form action="/messages" method="delete">
+      <p>
+        Name: <input type="text" name="from" placeholder="Your Name" /> <br />
+        Message: <input type="text" name="text" placeholder="The message..." />
+        <br />
+      </p>
+      <button type="submit">
+        Delete
+      </button>
+    </form>
```

Comment:
This doesn't connect to the delete method you have written in server.js. The reason it doesn't work is because you need to provide the id as a path parameter (as set out in your API URL here: https://github.com/CodeYourFuture/node-challenge-chat-server/pull/138/files#diff-a4c65ede64197e1a112899a68bf994485b889c4b143198bac4af53425b38406fR70).